src/Utils/Hooks.php

```diff
@@ -11,4 +11,6 @@ class Hooks
 	const AFTER_ORDER = 'ln_wc_after_order';
```

Comment:
I wonder if we shouldn't change this for an interface same with `ErrorCodes` if neither of those files are going to have a method.